lib/xMsExampleExtractor.js

```diff
@@ -57,29 +52,28 @@ class XMsExampleGenerator {
     }
 
     this.options = options;
+    log.debug(`specPath : ${this.specPath}`);
+    log.debug(`recordings : ${this.recordings}`);
+    log.debug(`options.output : ${this.options.output}`);
+    log.debug(`options.matchApiVersion : ${this.options.matchApiVersion}`);
   }
 
   /**
+   * Extracts x-ms-examples from the recordings
    */
-  generate() {
+  extract() {
     let self = this;
     self.mkdirSync(self.options.output);
     self.mkdirSync(self.options.output + "/examples");
-    self.mkdirSync(self.options.output + "/swagger");
```

Comment:
swagger needs to get generated, because it needs "x-ms-examples" with the reference to the proper file in there.